src/Profile.js

```diff
@@ -70,7 +70,7 @@ const Profile = ({
 
 	useEffect(() => {
 		if (walletData && walletData.account && !isLoading) {
-			onNeoBalance();
+			setTimeout(() => onNeoBalance(), 500);
```

Comment:
So if initialization takes longer it'll still fail?

Reply:
yes it's a synthetic timeout, but I checked on slow 3g, it was without error.

Reply:
Kludgy. But I don't have any better idea.